consensus/packets/bitarray.go

```diff
@@ -0,0 +1,91 @@
+/*
+ *    Copyright 2018 Insolar
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package packets
+
+import (
+	"encoding/binary"
+	"math"
+
+	"github.com/pkg/errors"
+)
+
+const sizeOfBlock = 8
+
+type bitArray struct {
+	array    []uint8
+	bitsSize uint
+}
+
+func newBitArray(size uint) *bitArray {
+	totalSize := uint64(math.Round(float64(size/sizeOfBlock) + 0.5))
```

Comment:
Could we extract `int(math.Round(float64(`**`x`**`/`**`y`**`) + 0.5 `  into function? This pattern is used multiple times across PR